.circleci/main-config.yml

```diff
@@ -715,7 +715,7 @@ commands:
           additional_options: '-Dexec.mainClass=org.broadinstitute.ddp.route.IntegrationTestSuite -Dexec.classpathScope=test'
           background: true
       - wait_for_server:
-          port: 5556
+          port: 5999 # we're waiting for the post-liquibase port to open up, not the main spark port
```

Comment:
See [corresponding java code change](https://github.com/broadinstitute/ddp-study-server/pull/2854/files#r1571285356).

Reply:
Thx for that comment!